### PR TITLE
Add dynamic popup

### PR DESCRIPTION
**Pull request by Samuelwanza**
refactored the grid items in index.html
coded grid-items in index,js
styled the grid-items and popup in index.css
completed mobile and desktop versions

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.css [hunk 1]
```diff
@@ -239,6 +239,101 @@ body {
...
+  margin-top: 5px;
+}
+
+.pop-up-menu-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (1GORDON) [inline index.css:263]: Great design team 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the Figma design as much as possible. Kindly ensure the buttons are well positioned on the page and they are not overflowing.
- Your design 
your-mobile-popup
- Given design
Given-mobile-popup

File: index.css [hunk 2]
```diff
@@ -752,4 +852,39 @@ textarea::placeholder {
...
   }
+
+  .pop-up-menu {
+    width: 70vw;
+    height: 90vh;
+  }
+
+  .pop-up-menu-wrapper {
+    display: flex;
+  }
```
Comment (1GORDON) [inline index.css:863]: Great design team🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the Figma design provided. I would recommend you fix your desktop popup image as it appears distorted and blurred at the moment.
- Your design
Your-desktop-image
- Given design
Given-image

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Samuelwanza & @rodrigopenayo98 ✋

Good job so far!

## Highlights!
- Followed GitHub Flow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Samuelwanza , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested changes. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**